src/components/Fields/AmountField.jsx

```diff
@@ -26,11 +26,13 @@ class AmountField extends React.Component {
 
 	componentDidMount() {
 		this.props.setDefaultAsset();
+		this.props.getTransferFee().then((fee) => fee && this.onFee(fee));
```

Comment:
can we use `await` here?

Reply:
`await` can pause componentDidMount execution